app/views/govuk_publishing_components/components/docs/layout_for_admin.yml

```diff
@@ -23,3 +23,19 @@ examples:
       browser_title: 'A page title'
       block: |
         <!-- You probably want to use the header, main & footer components here -->
+  with_custom_js_filename:
+    data:
+      environment: production
+      product_name: Publishing
+      browser_title: 'A page title'
+      js_filename: "application"
```

Comment:
These examples pass the default value, would it be clearer to show this functionality in action?

Reply:
The problem with that is that i'd have to add a blank `admin` stylesheet, because it will actually try to compile the file. So it if update it to use "admin" it'll look for "admin.js" and blow up if it can't find it.

I figured this was the lesser of 2 evils, but let me know if you'd prefer another approach 👍

Reply:
Oh I see that makes sense. Tricky! In that case maybe add a description to these examples to clarify? Something like:

```
description: An alternative JS filename can be used in place of the default `application.js` if required (note that this cannot easily be demonstrated here).
```

Reply:
great shout. Thanks Andy will sort that now